modules/test/playwright/pages/object-web/model-builder/ModelBuilderObjectDefinitionNodePage.ts

```diff
@@ -76,7 +76,8 @@ export class ModelBuilderObjectDefinitionNodePage {
 		this.objectFieldLabelInput = page
 			.locator('div.form-group')
 			.filter({hasText: /^LabelMandatory$/})
-			.getByRole('textbox');
+			.getByRole('textbox')
+			.nth(1);
```

Comment:
Hi @larissacribeiro, could we bring the modal reference to this locator?
Something like this:

```
this.modalAddObjectFieldLabelInput = this
	.modalAddObjectField
	.getByRole('textbox', {name: 'Label' + 'Mandatory'});
```